config/components.php

```diff
@@ -231,6 +232,19 @@
                     $item->searchHits  += $matches;
                     $item->searchScore += $matches * $score;
                 }
+
+                // check for similar matches
+                if ($options['words'] === false && $options['similar'] === true) {
+                    $matches = Str::similarity($lowerQuery, $lowerValue, $similarPercent);
+
+                    if ($matches > 0) {
+                        // calculate score based on percentage
+                        // for example 66.6% result: (66.6 / 10) * $score, final result is (7 * $score)
+                        // the coefficient can be a value between 0 and 10
+                        $item->searchScore += round($similarPercent / 10) * $score;
```

Comment:
Have you tested the real-world behavior of the fuzzy search? I.e. does it sort the results in a good and expected order?

In my opinion a multiplier of up to 10 is a bit high compared to the multipliers above. Even an exact beginning match only gets a score of `8 * $score`. I haven't tested this, but my guess would be that the similar matches end up too high in the results list.

Reply:
I will do some tests and share the results.